features/plugin-recently-active.feature

```diff
@@ -0,0 +1,103 @@
+Feature: List Recently Active WordPress plugins
```

Comment:
Can we rename this file to `plugin-list-recently-active.feature`?

Also, given we have a code path for multisite, we should have a multisite scenario too.

Reply:
Updated in https://github.com/wp-cli/extension-command/pull/383/commits/805760d37ffd2fc84aa4f7e874fcbf3944af9d79